plugins/admin/README.md

```diff
@@ -9,6 +9,8 @@ Plugin for generating api components schema from `AdminQuery` queries. This sche
   paths are relative to directory from your current CWD. Unless you are using JavaScript files - in that case you can
   use `__dirname` and `path.join`/`path.resolve` for paths relative to configuration file.
 
+  - `serverVersion` - version of the contracts generator that should be used. Default: `3.0.0`
```

Comment:
I think that we won't remember to update this default 😅

Reply:
I mean we won't update it in readme ;)

Reply:
Yeah, you are right